apps/web/src/hooks/useErrorMessages.ts

```diff
@@ -64,8 +64,9 @@ const useErrorMessages = (): ErrorWithMessages => {
   };
 
   if (isRouteErrorResponse(error) && "status" in error) {
+    const data = error.data as Error;
```

Comment:
It is pretty safe because of the `isRouteErrorResponse` but I think it is probably more safe after [update cast to be more accurate](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11719/commits/be937f073a3cc179a729d6202344e89c17a35356) :woman_shrugging:

Reply:
Oh yeah, nice.  :heart: